Classes/CalcsTab.lua

```diff
@@ -73,6 +73,14 @@ local CalcsTabClass = newClass("CalcsTab", "UndoHandler", "ControlHost", "Contro
 				self:AddUndoState()
 				self.build.buildFlag = true
 			end)
+		}, },{ label = "Skill Stages", playerFlag = "multiStage", { controlName = "mainSkillStageCount",
+			control = new("EditControl", nil, 0, 0, 52, 16, nil, nil, "%D", nil, function(buf)
```

Comment:
What do you think about making this a Slider (like for crafting item modifiers between a min/max range) instead of a integer counter widget?

Reply:
Having this as a slider would break existing builds.